### ISSUE TITLE
Submitting "Product Description Feedback" with screenshot is very slow and no indicator

### ISSUE THREAD

**Issue by rprakash20** [topcoder-archive/TCO21-Regionals-QA-Competition#47]
### Steps to Reproduce

1. Open https://www.newegg.com/ in browser
2. Open a product's page
3. Scroll down to "FEEDBACK" button in product overview section
4. Click on the FEEDBACK
5. Check "Include Screenshot"
6. Notice that the page screenshot will be added to the modal dialog
7. Notice the time it takes to load the screenshot
8. Click on Submit and notice the behavior

### Target URL
https://www.newegg.com/intel-core-i7-11700k-core-i7-11th-gen/p/N82E16819118233?quicklink=true

### Screenshots or Screen Capture

https://user-images.githubusercontent.com/5712602/128192880-0fb30573-66cf-4cb5-9b80-aec9ae06a86d.mov

### Current Results
Submitting "Product Description Feedback" with screenshot is very slow and there is no indicator it gives a feeling that it's broken.

### Expected Results
Submitting "Product Description Feedback" with screenshot should be faster and while its getting submitted indicator should be shown.

### Browser version and OS version
- Device: MacBook Pro 13 inch
- Browser: Version 92.0.4515.107 (Official Build) (x86_64)
- OS Version: macOS Catalina 10.15.4

**Comment by macs054** [comment]
Duplicate of #29

**Comment by rprakash20** [comment]
I think this is not a duplicate , since workflow is different and this is for product feedback, other one #29 is for overall feedback for the website.

**Comment by macs054** [comment]
You're right, but since it's the same functionality with the same results I'll let the reviewer decide.

**Comment by codejamtc** [comment]
The same function will be used in both places; there is a loading indication also

**Submitter**: 0 Points
**Challenger**: 8 Points
